src/Lykke.Job.BlockchainCashinDetector/Workflow/Sagas/CashinSaga.cs

```diff
@@ -367,6 +391,27 @@ private async Task Handle(DepositWalletLockReleasedEvent evt, ICommandSender sen
 
             var transitionResult = aggregate.OnDepositWalletLockReleased();
 
+            // Sending the "off-blockchain operation" event, if needed.
+            if (aggregate.IsDustCashin.Value &&
+                transitionResult.ShouldSendCommands())
```

Comment:
As you could see in another event handlers, aggregate should be saved before command sending